src/parser.rs

```diff
@@ -2680,19 +2685,8 @@ impl<'a> Parser<'a> {
             None
         };
 
-        // Not Sure if Top should be cheked here as well. Trino doesn't support TOP.
```

Comment:
this was added in #390  but is now covered by the general purpose `Tuple` parsing